scene/gui/gradient_edit.h

```diff
@@ -34,6 +34,7 @@
 #include "scene/gui/color_picker.h"
 #include "scene/gui/popup.h"
 #include "scene/resources/gradient.h"
+#include "editor/editor_node.h"
```

Comment:
Includes should be in cpp file when possible (this file has some wrong includes that shouldn't be there; let's not add more).

Reply:
I moved the include to the cpp file.